I added a script to build the conda environment to make testing and installation easier.  
When I tried to build the docs, I got an error:  

Theme error:
An error happened in rendering the page _autosummary/demo.
Reason: UndefinedError("'style' is undefined")
make: *** [html] Error 2

--- a/demo/requirements_demo.txt
+++ b/demo/requirements_demo.txt
@@ -1,8 +1,3 @@
-numpy~=1.20.0
-bm3d~=3.0.8
-Pillow~=8.1.0
-matplotlib~=3.3.4
-dotmap~=1.3.23
+-r ../requirements.txt
 scikit-image>=0.18
 imageio>=2.9.0
-pnp_mace

--- a/dev_scripts/clean_install_all.sh
+++ b/dev_scripts/clean_install_all.sh
@@ -2,8 +2,8 @@
 
 # Create and activate new conda environment
 
-ROOTPATH=$(realpath $(dirname $0)/..)
-
+ROOTPATH=$(realpath ..)
+echo $ROOTPATH
 # First check if the target environment is active and deactivate if so
 NAME=pnp_mace
 if [ "$CONDA_DEFAULT_ENV"==$NAME ]; then
@@ -15,10 +15,10 @@ conda remove env --name $NAME --all -y
 conda create --name $NAME python=3.10 -y
 conda activate $NAME
 
-pip install -r $ROOTPATH/docs/requirements.txt
-pip install $ROOTPATH
+pip install -r "$ROOTPATH/docs/requirements.txt"
+pip install "$ROOTPATH"
 
-cd $ROOTPATH/docs
+cd "$ROOTPATH/docs"
 /bin/rm -rf _build
 
 make clean html

--- a/dev_scripts/clean_install_all.sh
+++ b/dev_scripts/clean_install_all.sh
@@ -1,6 +1,9 @@
 #!/bin/bash
 
 # Create and activate new conda environment
+
+ROOTPATH=$(realpath $(dirname $0)/..)
+
 # First check if the target environment is active and deactivate if so
 NAME=pnp_mace
 if [ "$CONDA_DEFAULT_ENV"==$NAME ]; then
@@ -8,25 +11,22 @@ if [ "$CONDA_DEFAULT_ENV"==$NAME ]; then
 fi
 
 # Then remove the old version and reinstall
-yes | conda remove env --name $NAME --all
-yes | conda create --name $NAME python=3.9
+conda remove env --name $NAME --all -y
+conda create --name $NAME python=3.10 -y
 conda activate $NAME
-pip install -r ../requirements.txt
-pip install ..
 
-pip install -r ../demo/requirements_demo.txt
+pip install -r $ROOTPATH/docs/requirements.txt
+pip install $ROOTPATH
 
-cd ../docs
-/bin/rm -r build
+cd $ROOTPATH/docs
+/bin/rm -rf _build
 
-pip install -r requirements.txt
 make clean html
 
-echo ""
-echo "*** The html documentation is at pcdrecon/docs/build/html/index.html ***"
-echo ""
-
-cd ../dev_scripts
 echo
 echo "Use 'conda activate" $NAME "' to activate this environment."
 echo
+
+echo ""
+echo "*** The html documentation is at PnP-MACE/docs/_build/html/index.html ***"
+echo ""

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -1,13 +1,7 @@
-numpy~=1.20.0
-bm3d~=3.0.8
+-r ../demo/requirements_demo.txt
 setuptools~=53.0.0
-Pillow~=8.1.0
-matplotlib~=3.3.4
-dotmap~=1.3.23
 requests~=2.25.1
-Sphinx>=3.4
-sphinx_rtd_theme~=0.5.1
-sphinx_autodoc_typehints~=1.11.0
-sphinxcontrib-bibtex~=2.1.4
-scikit-image>=0.18
-imageio>=2.9.0
+sphinx>=7.0
+sphinx_rtd_theme
+sphinx_autodoc_typehints
+sphinxcontrib-bibtex

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
-numpy~=1.20.0
-bm3d~=3.0.8
-Pillow~=8.1.0
-matplotlib~=3.3.4
+numpy
+bm3d>=4.0.1
+Pillow>=8.1.0
+matplotlib>=3.3
 dotmap~=1.3.23
 requests~=2.25.1

